build/buildconstants/params_mainnet.go

```diff
@@ -113,22 +122,19 @@ const UpgradeWaffleHeight abi.ChainEpoch = 4154640
 // var because of TestMigrationNV24 in itests/migration_test.go to test the FIP-0081 pledge ramp
 var UpgradeTuktukHeight abi.ChainEpoch = 4461240
 
-// ??????
-var UpgradeTeepHeight = abi.ChainEpoch(9999999999)
-
 // FIP-0081: for the power actor state for pledge calculations.
 // UpgradeTuktukPowerRampDurationEpochs ends up in the power actor state after
 // Tuktuk migration. along with a RampStartEpoch matching the upgrade height.
 var UpgradeTuktukPowerRampDurationEpochs = uint64(builtin.EpochsInYear)
 
-// This fix upgrade only ran on calibrationnet
-const UpgradeWatermelonFixHeight abi.ChainEpoch = -1
+// ??????
+var UpgradeTeepHeight = abi.ChainEpoch(9999999999)
 
-// This fix upgrade only ran on calibrationnet
-const UpgradeWatermelonFix2Height abi.ChainEpoch = -2
+// This epoch, 90 days after Teep is the completion of FIP-0100 where actors will start applying
+// the new daily fee to pre-Teep sectors being extended.
+var UpgradeTockHeight = UpgradeTeepHeight + builtin.EpochsInDay*90
 
-// This fix upgrade only ran on calibrationnet
-const UpgradeCalibrationDragonFixHeight abi.ChainEpoch = -3
+var UpgradeTeepInitialFilReserved = InitialFilReserved // FIP-0100: no change for mainnet
```

Comment:
should also be constants on mainnet.

Reply:
can't because `InitialFilReserved` isn't, because we calculate it from whole FIL

Reply:
Ah, I see. But what about the heights?

Reply:
no, because we have `LOTUS_DISABLE_TEEP` set; but I assume that once we bump to nv27 that we can go back and make those consts, maybe that's on the checklist? @rjan90 ?

Reply:
AH.